src/js/components/recipient/RecipientOverview.jsx

```diff
@@ -6,7 +6,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { CaretRight } from 'components/sharedComponents/icons/Icons';
+import { CaretRight, InfoCircle } from 'components/sharedComponents/icons/Icons';
```

Comment:
We've not deleted this code yet, but we're migrating to font awesome. All Icons moving foward, to my understanding, should be imported from the releavnt `@fortawesome` package

Reply:
Done.